<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>workwindow</class>
 <widget class="QWidget" name="workwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">/*-----QWidget-----*/
QWidget
{
	background-color: #DEF2F1;
	color: #000000;
	border-color: #000000;

}

/*-----QPushButton-----*/
QPushButton
{
	background-color: #379683;
	color: rgb(237, 245, 225);
    font-family: &quot;Arial&quot;;
    font-size: 14px;
	border-style: solid;
	border-width: 2px;
	border-radius: 6px;
	border-color: #379683;
	margin: 2px;

}
QPushButton {
    background-color: #379683;
    color: rgb(237, 245, 225);
    font-family: &quot;Arial&quot;;
    font-size: 14px;
    border-style: solid;
    border-width: 2px;
    border-radius: 6px;
    border-color: #379683;
    margin: 2px;
}

QPushButton:hover {
    background-color: #2d6a5f; 
}

QPushButton:pressed {
    background-color: #266057;
}

/*-----QLineEdit-----*/
QLineEdit
{
    font-family: &quot;Arial&quot;;
	background-color: rgb(237, 245, 225);
	color: #000000;
	border-style: solid;
	border-width: 1px;
	border-radius: 4px;
	border-color: #000000;
	font-family: &quot;Azonix&quot;;

}

QToolButton {
    background-color: transparent; 
    border: none; 
}

QLabel {
    font-family: &quot;Arial&quot;;
}</string>
  </property>
  <widget class="QFrame" name="WorkFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1081</width>
     <height>701</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QStackedWidget" name="WorkPages">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1061</width>
      <height>681</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="NotAccessPage">
     <widget class="QLabel" name="NoAccessLabel">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>260</y>
        <width>421</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>44</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>НЕТ ДОСТУПА</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ExitButton_NoAcc">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>640</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>-1</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Выйти</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="CurrectUserLineEdit_NoAcc">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>30</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="CurrectUserLabel_NoAcc">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>10</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Текущий пользователь</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="AccessPage">
     <widget class="QComboBox" name="CurrectTableCmB_Work">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>540</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ExitButton_Work">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>640</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Выйти</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SaveButton_Work">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>640</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
     <widget class="QLabel" name="CurrectTable_Work">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>520</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Текущая таблица</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="CurrectUserLineEdit_Work">
      <property name="geometry">
       <rect>
        <x>832</x>
        <y>60</y>
        <width>221</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="CurrectUserLabel_Work">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>40</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Текущий пользователь</string>
      </property>
     </widget>
     <widget class="QToolButton" name="ReloadButtonIcon_Work">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>520</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>660</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="InfoLabel_Work">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>640</y>
        <width>241</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="DelRowBut_Work">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>530</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="AddRowBut_Work">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>530</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SortTableBut_Work">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>530</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Отсортировать</string>
      </property>
     </widget>
     <widget class="QLabel" name="SortLabel_Work">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>580</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Сортировать по</string>
      </property>
     </widget>
     <widget class="QComboBox" name="SortCmB_Work">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>600</y>
        <width>211</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="MainTable_Work">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>1051</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Logo_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>800</width>
        <height>70</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="AdminPage">
     <widget class="QGroupBox" name="CreateUser">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>331</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>Создать пользователя</string>
      </property>
      <widget class="QLineEdit" name="LogLineEdit_Adm">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="PassLineEdit_Adm">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>201</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="LogLabel_Adm">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Логин</string>
       </property>
      </widget>
      <widget class="QLabel" name="PassLabel_Adm">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>49</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Пароль</string>
       </property>
      </widget>
      <widget class="QLabel" name="AccessLevelLabel_Adm">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Уровень доступа</string>
       </property>
      </widget>
      <widget class="QComboBox" name="AccessLevelCmB1_Adm">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>130</y>
         <width>31</width>
         <height>22</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>7</x>
         <y>180</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="CreateUserButton_Adm">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>160</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Создать</string>
       </property>
      </widget>
      <widget class="QLabel" name="InfoLabel_CreateUserAdm">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>191</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="TableControl">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>10</y>
        <width>681</width>
        <height>661</height>
       </rect>
      </property>
      <property name="title">
       <string>Управление таблицами</string>
      </property>
      <widget class="QLabel" name="CurrectTableLabel_Adm">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>470</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Текущая таблица</string>
       </property>
      </widget>
      <widget class="QComboBox" name="CurrectTableCmB_Adm">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>470</y>
         <width>181</width>
         <height>22</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
      </widget>
      <widget class="QPushButton" name="SaveButton_Adm">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>610</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
      <widget class="QToolButton" name="ReloadButtonIcon_Adm">
       <property name="geometry">
        <rect>
         <x>641</x>
         <y>470</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="DelRowBut_Adm">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>610</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
      <widget class="QPushButton" name="AddRowBut_Adm">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>610</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
      <widget class="QPushButton" name="SortTableBut_Adm">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>610</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>Отсортировать</string>
       </property>
      </widget>
      <widget class="QComboBox" name="SortCmB_Adm">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>510</y>
         <width>181</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="SortLabel_Adm">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>510</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Сортировать по</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="MainTable_Adm">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>661</width>
         <height>441</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="ExitButton_Adm">
       <property name="geometry">
        <rect>
         <x>540</x>
         <y>560</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Выйти</string>
       </property>
      </widget>
      <widget class="QLabel" name="Logo">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>469</y>
         <width>340</width>
         <height>60</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="EditUser">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>331</width>
        <height>431</height>
       </rect>
      </property>
      <property name="title">
       <string>Выдать права</string>
      </property>
      <widget class="QListView" name="UsersList_Adm">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>30</y>
         <width>161</width>
         <height>311</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="AccessLevelCmBox2_Adm">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>31</width>
         <height>22</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="AccessLevelLabel_Adm_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Уровень доступа</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ChangeAccessLevelButton_Adm">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>390</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Заменить</string>
       </property>
      </widget>
      <widget class="QLabel" name="InfoLabel_Adm">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>351</y>
         <width>311</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="NewPassLineEdit_Adm">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>141</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="NewPassLabel_Adm">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Новый пароль</string>
       </property>
      </widget>
      <widget class="Line" name="line_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="DelUserButton_Adm">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>390</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>370</y>
         <width>311</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="SortUsersButton_Adm">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>310</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Сортировать</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
